internal/services/monitor/monitor_activity_log_alert_resource.go

```diff
@@ -466,6 +470,9 @@ func resourceMonitorActivityLogAlertRead(d *pluginsdk.ResourceData, meta interfa
 	d.Set("resource_group_name", id.ResourceGroupName)
 
 	if model := resp.Model; model != nil {
+
+		d.Set("location", location.Normalize(*model.Location))
+
```

Comment:
Can we change this to 
```suggestion
		d.Set("location", location.NormalizeNilable(model.Location))
```